src/app/locales/fr/routes.yml

```diff
@@ -155,13 +155,13 @@ user:
             sort-by-login: Trier par la date de dernière connexion
             sort-desc: Trier par ordre décroissant
             sort-asc: Trier par ordre croissant
-        empty: La liste retournée des chaînes suivie est vide.
+        empty: La liste retournée des chaînes suivies est vide.
 search:
     header:
-        all: Tous les résultat de la recherche
+        all: Tous les résultats de la recherche
         games: "Résultat de la recherche de jeux: "
         channels: "Résultat de la recherche de chaînes: "
-        streams: "Résultat de la recherche de streams: "
+        streams: "Résultat de la recherche de streams : "
```

Comment:
The other lines would need to be changed here as well if you change the `search.header.streams` headline.